components/Tabs/src/TabBarView/MDCTabBarView.h

```diff
@@ -36,6 +37,11 @@ __attribute__((objc_subclassing_restricted)) @interface MDCTabBarView : UIScroll
 /** The tab bar view delegate. */
 @property(nullable, nonatomic, weak) id<MDCTabBarViewDelegate> tabBarDelegate;
 
+/**
+ The template for the selection indicator.
+ */
+@property(nonnull, nonatomic, strong) id<MDCTabBarViewIndicatorTemplate> selectionIndicatorTemplate;
```

Comment:
Nit: Could be easier to understand if had a short explanation of what template means in this context (unless already explained elsewhere?)

Reply:
Added a bit more. If we need further details, it should probably be in a follow-up PR.